Content.Shared/Backmen/Surgery/SharedSurgerySystem.Steps.cs

```diff
@@ -197,7 +197,7 @@ private void OnTendWoundsStep(Entity<SurgeryTendWoundsEffectComponent> ent, ref
             || !group.Any(damageType => damageable.Damage.DamageDict.TryGetValue(damageType, out var value)
                 && value > 0)
             && (!TryComp(args.Part, out BodyPartComponent? bodyPart)
-            || bodyPart.Integrity == BodyPartComponent.MaxIntegrity))
+            || bodyPart.Integrity <= BodyPartComponent.MaxIntegrity))
```

Comment:
Looks like I forgor to invert the operator, will fix that

Reply:
> Need to Cherry Pick this PR, but won't this always be true if we're trying to heal the entity? The fact you're trying to heal it means that the integrity WILL be lower than the maximum.

Actually Maximum is a constant 0 now, but theoretically it can be changed to whatever you want. There's a `ClampMin` function that will make all damages At Least equal to MaxIntegrity, so it should always work correctly.

And the lower sign is here to prevent the stupid floating point rounding shit warning